### DISCUSSION TITLE
Requirement to have access to copilot not mentioned in plugins

### DISCUSSION THREAD

**Discussion by mooreds** [community/community#6858] [Copilot Conversations] [answered]
I filed a PR here: https://github.com/github/copilot.vim/pull/2

but it appears that is the incorrect way to give feedback.

I for one am super excited that GitHub is supporting more editors. But it'd be great if each of the install instructions made it clear that you needed to have access to Copilot. I went through the entire install process and then was instructed that I needed to join a waitlist. From the feedback on the PR, I am not alone in that.

Thanks again for a great tool, looking forward to exploring it when I'm off the waitlist :).

**Comment by tpope** [comment, marked as answer]
Updated the README, thanks for letting us know!
